internal/consensus/invalid_test.go

```diff
@@ -80,12 +79,14 @@ func invalidDoPrevoteFunc(t *testing.T, cs *State, sw *p2p.Switch, pv types.Priv
 
 		// precommit a random block
 		blockHash := bytes.HexBytes(cmtrand.Bytes(32))
+		timestamp := cs.voteTime(cs.Height)
+
 		precommit := &types.Vote{
 			ValidatorAddress: addr,
 			ValidatorIndex:   valIndex,
 			Height:           cs.Height,
 			Round:            cs.Round,
-			Timestamp:        cmttime.Now(),
+			Timestamp:        timestamp,
```

Comment:
Does this change really matter?

Reply:
Hmm, I'm guessing @lasarojc brought it because it was a change that the forward-ported code had been doing. If that's the case, I'm OK leaving this change, as it'll disappear when merging back to `main`

Reply:
I am reverting here to keep the feature branch simpler (As @sergio-mena mentioned, this was forward-ported and I am reverting with as little further changes as possible).